parsers/SE.py

```diff
@@ -0,0 +1,77 @@
+import datetime
+from collections import defaultdict
+
+import arrow
+import requests
+
+SVK_URL = 'http://www.svk.se/ControlRoom/GetProductionHistory/' \
+          '?productionDate={date}&countryCode={zoneKey}'
+
+# what the value refer to (FYI)
+NAMES_MEANING = {
+    2: 'nuclear',
+    4: 'thermal',
+    5: 'wind',
+    6: 'unknown',
+    1: 'production',
+    7: 'consumption',
+    3: 'hydro'
+}
+# mapping from the value to electricity map values
+MAPPING = {
+    2: 'nuclear',
+    4: 'unknown',
+    5: 'wind',
+    6: 'unknown',
+    1: None,
+    7: None,
+    3: 'hydro'
+}
+
+
+def fetch_production(zone_key='SE', session=None, target_datetime=None,
+                     logger=None):
+    # parse target_datetime - and convert None to now
+    target_datetime = arrow.get(target_datetime).datetime
+    url = SVK_URL.format(date=target_datetime.strftime('%Y-%m-%d'),
+                         zoneKey='SE')
+
+    data = requests.get(url).json()
+
+    # creating a dict looking like
+    # {'2018-01-01 01:00': {'nuclear': xxx, 'hydro': ...}, ...}
+    productions = defaultdict(lambda: defaultdict(lambda: 0))
+    for sub_data in data:
+        # sub_data is a list of (x, y) values for that prod type
+        # x is a timestamp, y a value (MWH)
+        name = MAPPING[sub_data['name']]
+        if not name:
+            continue
```

Comment:
I think we should issue a warning using our logger here.

Reply:
`MAPPING` intentionally maps two entries (corresponding to the total production and total consumption) to None. If it's another entry, it will raise a `KeyError`.

Reply:
got it! I was still in Javascript world :)